jobs/cc_route_syncer/templates/cloud_controller_ng.yml.erb

```diff
@@ -62,6 +62,18 @@ database_encryption:
   pbkdf2_hmac_iterations: <%= link("cloud_controller_internal").p("cc.database_encryption.experimental_pbkdf2_hmac_iterations") %>
 <% end %>
 
+<% if_link("cloud_controller_container_networking_info") do |info|
+  internal_vip_range = "#{info.p("cc.internal_route_vip_range")}"
+  raise StandardError.new("invalid cc.internal_route_vip_range: #{internal_vip_range}") unless internal_vip_range =~ /\A (?:\d{1,3}\.){3} \d{1,3} \/ \d{1,3} \z/x
```

Comment:
Can you add some template tests for this behavior?

https://github.com/cloudfoundry/capi-release/blob/develop/spec/cloud_controller_ng/cloud_controller_ng_spec.rb

Reply:
Added!